src/main/java/org/mockito/internal/junit/NoOpTestListener.java

```diff
@@ -2,7 +2,7 @@
 
 import org.mockito.mock.MockCreationSettings;
 
-class NoOpTestListener implements MockitoTestListener {
+public class NoOpTestListener implements MockitoTestListener {
```

Comment:
yes. We still have 2 internal packages: internal.runners and internal.junit. We should merge them but perhaps after we stop maintaining 2.x?